script.speedfaninfo/default.py

```diff
@@ -0,0 +1,330 @@
+import datetime, os
```

Comment:
usually a good practice to make this file as small as possible. Maybe move this into `/resources/lib` and from the entryfile just import and run?